### ISSUE TITLE
Improve Text Annotator Workflow / Entry points

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#993]
I think we could be doing more to promote Text Annotator. The only entry point to it is in the "Analyze" menu.

Consider the following:
- [ ] automatic diversion to Text Annotator if the site search query string is either:
  - [ ] greater than x tokens
  - [ ] no results
- [ ] add an "annotate text" image to carousel on the homepage

@nlwashington @mellybelly @cmungall thoughts?

**Comment by cmungall** [comment]
+1 for both but dependent on #925 or we're back in a worse place w.r.t species

**Comment by jmcmurry** [comment]
@cmungall very good point. I will therefore postpone this to wishlist.
